setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Standard build script.
```

Comment:
Whats the process for building and releasing this library now? Be good to document it, esp for python noobs like me ;) Could be documented in this repo or on https://github.com/zaproxy/zaproxy/wiki/GeneratingTheFullRelease#generate-the-release-add-ons